docs/rpc.md

```diff
@@ -65,9 +69,80 @@ which would yield the response:
 | `sendrawtransaction` |
 | `submitblock` |
 | `validateaddress` |
+| `verifyproof` |
 
 #### Implementation notices
 
+##### `getaccountstate`
+
+The order of assets in `balances` section may differ from the one returned by
+C# implementation. Assets can still be identified by their hashes so it
+shouldn't be an issue.
+
+##### `getapplicationlog`
+
+Error handling for incorrect stack items differs with C# implementation. C#
+implementation substitutes `stack` and `state` arrays with "error: recursive
+reference" string if there are any invalid items. NeoGo never does this, for
+bad `state` items it uses byte array susbstitute with message "bad
+notification: ..." (may vary depending on the problem), for incorrect `stack`
+items it just omits them (still returning valid ones).
+
+##### `getassetstate`
+
+It returns "NEO" for NEO and "NEOGas" for GAS in the `name` field instead of
+language-aware JSON structures.
+
+##### `getblock` and `getrawtransaction`
+
+In their verbose outputs neo-go can omit some fields with default values for
+transactions, this includes:
+ * zero "nonce" for Miner transactions (usually nonce is not zero)
+ * zero "gas" for Invocation transactions (most of the time it is zero).
+ * empty strings in contract fields of Publish transactions
+ * zero precision/amount and empty name fields of Register transactions
+
+##### `getclaimable`
+
+`claimable` array ordering differs, neo-go orders entries there by the
+`end_height` field, while C# implementation orders by `txid`.
+
+##### `getcontractstate`
+
+C# implementation doesn't return `Payable` flag in its output, neo-go has
+`is_payable` field in `properties` for that.
+
+##### `getnep5transfers`
+
+`received` and `sent` entries are sorted differently, C# node uses
+chronological order and neo-go uses reverse chronological order (which is
+important for paging support, see Extensions section down below).
+
+##### `getrawmempool`
+
+neo-go doesn't support boolean parameter to `getrawmempool` for unverified
```

Comment:
We can still return only verified transactions for any bool parameter, because returned JSON structure is different when this parameter is provided and is true. Don't know if this is actually needed.

Reply:
I doubt that and even in C# case that's mostly useful for debugging, if you care about some particular transaction presence in the mempool you look for verified transactions anyway.